docs/EN_US/Installing_and_RunningTheHPCCPlatform/Installing_and_RunningTheHPCCPlatform.xml

```diff
@@ -2040,7 +2040,7 @@ sudo /etc/init.d/hpcc-init -c esp start
         <title>HPCC Systems systemd services</title>
 
         <para>HPCC Systems is extending support and development to more
-        systemd services. We intend to continue support for Cent OS 6 and
+        systemd services. We intend to continue support for older Cent OS and
```

Comment:
Is this true? Are we still supporting older OS versions and SystemV? Ask @Michael-Gardner to review this PR

Reply:
@Michael-Gardner please review